default_settings.json

```diff
@@ -0,0 +1,5254 @@
+
+	// Controls whether selections should have rounded corners.
+	"editor.roundedSelection": true,
+
+	// Render vertical rulers after a certain number of monospace characters. Use multiple values for multiple rulers. No rulers are drawn if array is empty.
+	"editor.rulers": [],
+
+	// Controls the visibility of the horizontal scrollbar.
+	//  - auto: The horizontal scrollbar will be visible only when necessary.
+	//  - visible: The horizontal scrollbar will always be visible.
+	//  - hidden: The horizontal scrollbar will always be hidden.
+	"editor.scrollbar.horizontal": "auto",
+
+	// The height of the horizontal scrollbar.
+	"editor.scrollbar.horizontalScrollbarSize": 12,
+
+	// Controls whether clicks scroll by page or jump to click position.
+	"editor.scrollbar.scrollByPage": false,
+
+	// Controls the visibility of the vertical scrollbar.
+	//  - auto: The vertical scrollbar will be visible only when necessary.
+	//  - visible: The vertical scrollbar will always be visible.
+	//  - hidden: The vertical scrollbar will always be hidden.
+	"editor.scrollbar.vertical": "auto",
+
+	// The width of the vertical scrollbar.
+	"editor.scrollbar.verticalScrollbarSize": 14,
+
+	// Controls the number of extra characters beyond which the editor will scroll horizontally.
+	"editor.scrollBeyondLastColumn": 5,
+
+	// Controls whether the editor will scroll beyond the last line.
+	"editor.scrollBeyondLastLine": true,
+
+	// Scroll only along the predominant axis when scrolling both vertically and horizontally at the same time. Prevents horizontal drift when scrolling vertically on a trackpad.
+	"editor.scrollPredominantAxis": true,
+
+	// Controls whether the editor should highlight matches similar to the selection.
+	"editor.selectionHighlight": true,
+
+	// Controls whether the semanticHighlighting is shown for the languages that support it.
+	//  - true: Semantic highlighting enabled for all color themes.
+	//  - false: Semantic highlighting disabled for all color themes.
+	//  - configuredByTheme: Semantic highlighting is configured by the current color theme's `semanticHighlighting` setting.
+	"editor.semanticHighlighting.enabled": "configuredByTheme",
+
+	// Overrides editor semantic token color and styles from the currently selected color theme.
+	"editor.semanticTokenColorCustomizations": {},
+
+	// Controls strikethrough deprecated variables.
+	"editor.showDeprecated": true,
+
+	// Controls when the folding controls on the gutter are shown.
+	//  - always: Always show the folding controls.
+	//  - mouseover: Only show the folding controls when the mouse is over the gutter.
+	"editor.showFoldingControls": "mouseover",
+
+	// Controls fading out of unused code.
+	"editor.showUnused": true,
+
+	// Whether leading and trailing whitespace should always be selected.
+	"editor.smartSelect.selectLeadingAndTrailingWhitespace": true,
+
+	// Controls whether the editor will scroll using an animation.
+	"editor.smoothScrolling": false,
```

Comment:
I think this should be `true`. I'm surprised it's not by default. Need to make sure it doesn't cause performance issues on e.g. Chromebooks.